bin/gatherer.py

```diff
@@ -247,9 +252,20 @@ def main():
             LOGGER.error("No valid config item provided")
             return
 
+    if opts.publish_port:
+        publish_port = int(opts.publish_port)
+    else:
+        publish_port = 0
+
+    if opts.nameservers:
+        publisher_nameservers = opts.nameservers.split(',')
+    else:
+        publisher_nameservers = None
```

Comment:
Can we use the argparse mechanism to do this instead ? (provide the type, default value, etc)